src/CSharp.Workspaces/CSharp/SyntaxLogicalInverter.cs

```diff
@@ -208,6 +208,23 @@ internal static SyntaxLogicalInverter GetInstance(Document document)
                 {
                     return DefaultInvert(expression);
                 }
+            case SyntaxKind.CoalesceExpression:
+            {
+                var binaryExpression = (BinaryExpressionSyntax)expression;
+                if (binaryExpression.Right.Kind() == SyntaxKind.FalseLiteralExpression)
+                {
+                    // !(x ?? false) === (x != true)
+                    return BinaryExpression(SyntaxKind.NotEqualsExpression, binaryExpression.Left, LiteralExpression(SyntaxKind.TrueLiteralExpression));
```

Comment:
`CSharpFactory` provides some helper methods to make these calls shorter: `CSharpFactory.NotEqualsExpression` and `CSharpFactory.TrueLiteralExpression`.